src/AirshipBindings.iOS.Automation/ApiDefinitions.cs

```diff
@@ -487,7 +489,6 @@ interface UAActionAutomation
         // + (null_unspecified instancetype)shared;
         [Static]
         [Export("shared")]
-        [New]
```

Comment:
Why did the `[New]` annotation get removed? For the life of me I can't find any documentation on `[New]`. If you can drop me a URL, I'd appreciate it.

Reply:
`[New]` is for constructors or factory methods. These are accessors, so they don't need it. You'll see that UAComponent doesn't have it in its Shared method, which these derive from.

Reply:
Since these were added by hand, I think the `[New]` just kind of slipped in there

Reply:
https://stackoverflow.com/questions/6576206/what-is-the-difference-between-the-override-and-new-keywords-in-c/6576212#6576212

Reply:
It makes a warning go away, but otherwise doesn't do anyting

Reply:
I'll add them back in. We'll need to update Xamboni to use them when reconstructing inherited factory methods.